Makefile

```diff
@@ -57,10 +58,14 @@ sync:
 	go mod download
 
 .PHONY: clean
-clean:
+clean: clean-config
 	go clean -i ./...
 	rm -rf $(BIN) $(DIST) $(HUGO)
 
+.PHONY: clean-config
+clean-config:
+	rm -rf $(CONFIG)
```

Comment:
your config is a file, not a folder. I personally find the chose of naming confusing; there is a config folder in the root of the project, but the config you're referring to here is the `identifier-registration.yaml` file, so the flags `r` on the remove command is not necessary.

Is the intention when cleaning config to remove the entire `config` folder?

Reply:
this is intended. This way `$(CONFIG)` can be used to add and delete more config files, but leave the config folder in place, because it contains example configs